alacritty/src/input.rs

```diff
@@ -246,6 +246,55 @@ impl<T: EventListener> Execute<T> for Action {
             Action::Mouse(MouseAction::ExpandSelection) => ctx.expand_selection(),
             Action::SearchForward => ctx.start_search(Direction::Right),
             Action::SearchBackward => ctx.start_search(Direction::Left),
+            Action::NextPrompt => {
+                let terminal = ctx.terminal_mut();
+
+                let point = if terminal.mode().contains(TermMode::VI) {
+                    terminal.vi_mode_cursor.point
+                } else {
+                    let display_offset = terminal.grid().display_offset() as i32;
+                    // If we're outside of Vi mode use top most visible line.
+                    Point::new(Line(terminal.screen_lines() as i32 - 1 - display_offset), Column(0))
+                };
+
+                let prompt_point = match terminal.next_prompt(point) {
+                    Some(point) => point,
+                    None => return,
+                };
+
+                if terminal.mode().contains(TermMode::VI) {
+                    terminal.vi_goto_point(prompt_point);
+                } else {
+                    let scroll = Scroll::Delta(point.line.0 - prompt_point.line.0);
+                    terminal.scroll_display(scroll);
+                }
+
+                ctx.mark_dirty();
```

Comment:
Might want to make this a function on `Term`? I think that would be more consistent with the rest of our code.